openai/tests/asyncio/test_endpoints.py

```diff
@@ -14,10 +14,12 @@
 # FILE TESTS
 async def test_file_upload():
     result = await openai.File.acreate(
-        file=io.StringIO(json.dumps({"text": "test file data"})),
-        purpose="search",
+        file=io.StringIO(
+            json.dumps({"prompt": "test file data", "completion": "tada"})
+        ),
+        purpose="fine-tune",
     )
-    assert result.purpose == "search"
+    assert result.purpose == "fine-tune"
     assert "id" in result
 
```

Comment:
@Andrew-Chen-Wang this test was failing, i just copied the test case from the synchronous version to fix it.